slackClient.js

```diff
@@ -0,0 +1,119 @@
+const slackEventsApi = require('@slack/events-api');
+const Entities = require('html-entities').AllHtmlEntities;
+const { MyAnimeList } = require('./data');
+const { BOT_ID } = require('./util/constants');
+
+/**
+ * Class that handles slack event subscription and post message responses
+ */
+class SlackClient {
+  /**
+   * @param {Object} params
+   */
+  constructor(params) {
+    this.events = params.slackEvents;
+    this.slack = params.slackWebClient;
```

Comment:
What do you think about instantiating the slack client inside this module (but outside of this class) ? That way we can keep all slack-logic in one file